pennylane/ops/qubit/qchem_ops.py

```diff
@@ -0,0 +1,640 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This submodule contains the discrete-variable quantum operations that come
+from quantum chemistry applications.
+"""
+import cmath
+
+# pylint:disable=abstract-method,arguments-differ,protected-access
+import math
+import numpy as np
+
+import pennylane as qml
+from pennylane.operation import Operation
+
+from .non_parametric_ops import CNOT
+from .parametric_ops import RX
+
+INV_SQRT2 = 1 / math.sqrt(2)
+
+# Four term gradient recipe for controlled rotations
+c1 = INV_SQRT2 * (np.sqrt(2) + 1) / 4
+c2 = INV_SQRT2 * (np.sqrt(2) - 1) / 4
+a = np.pi / 2
+b = 3 * np.pi / 2
+four_term_grad_recipe = ([[c1, 1, a], [-c1, 1, -a], [-c2, 1, b], [c2, 1, -b]],)
+
+
+class IsingXX(Operation):
```

Comment:
I'm not sure I personally would associate the Ising operations as qchem. I think they belong in the parametric file more so. Maybe worth checking in the team to find its best place.

Also, the Ising ops are special cases to `qml.PauliRot` which is in the parametric file.